code/components/extra-natives-five/src/NuiAudioSink.cpp

```diff
@@ -1074,14 +1074,15 @@ namespace rage
 		static audCategoryControllerManager* GetInstance();
 	};
 
+	static audCategoryControllerManager** g_audCategoryControllerManager = nullptr;
 	audCategoryControllerManager* audCategoryControllerManager::GetInstance()
 	{
-#ifdef GTA_FIVE
-		static auto patternRef = hook::get_address<audCategoryControllerManager**>(hook::get_pattern("45 33 C0 BA 90 1C E2 44 E8", -4));
-#elif IS_RDR3
-		static auto patternRef = hook::get_address<audCategoryControllerManager**>(hook::get_pattern("48 8B 0D ? ? ? ? E8 ? ? ? ? 8B 15 ? ? ? ? 48 8D 0D ? ? ? ? E8", 3));
-#endif
-		return *patternRef;
+		if (!g_audCategoryControllerManager)

```

Comment:
`audCategoryControllerManager::GetInstance` being called from another HookFunction prior to this pattern running a bit nasty. Any potential side-effects here (i.e., until the first `Mumble_RunFrame` call)? I don't believe so, but a second opinion would not hurt.

Reply:
I believe this is only called here?

https://github.com/citizenfx/fivem/blob/master/code/components/extra-natives-five/src/NuiAudioSink.cpp#L2878

Should be OK. Fixed the merge conflicts anyway.